Phonebook.Frontend/src/app/services/api/organigram.service.ts

```diff
@@ -62,6 +63,33 @@ export class OrganigramService {
       node.employees.push(person);
     }
   }
+
+  public getNode(): Observable<UnitTreeNode | null | undefined> {
+    return combineLatest([this.currentUserService.getCurrentUser(), this.getOrganigram()]).pipe(
+      map(([user, organigram]) => {
+        if (user === null) {
+        return;
```

Comment:
@DanielHabenicht is this good? I think it is better to throw here an exception.

Reply:
there should be a return null, that way the definition of the return values in the function can be condensed to `Observable<UnitTreeNode | null>`